src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1209,7 +1209,7 @@ private void updateOverallHappiness()
             final BuildingHome home = citizen.getHomeBuilding();
             if(home != null)
             {
-                housing = home.getBuildingLevel();
+                housing += home.getBuildingLevel();
```

Comment:
@xavierh ahhh, my understanding of the level system/happiness system is what was in error here..  
I thought: 
```  
1 citizen in level 1 house = 1 happy.. 
1 citizen in level 2 house = 2 happy..
```
Not that that a house of level 2 could hold 2 people.  
That clears up that confusion.  
How do you count extra houses if we are iterating through citizens, but we have more houses than citizens? (Eg, a citizen hasn't spawned yet, or one was killed).
There would potentially have a house that is not counted. @xavierh ahhh, my understanding of the level system/happiness system is what was in error here..  
I thought: 
```  
1 citizen in level 1 house = 1 happy.. 
1 citizen in level 2 house = 2 happy..
```
Not that that a house of level 2 could hold 2 people.  
That clears up that confusion.  
How do you count extra houses if we are iterating through citizens, but we have more houses than citizens? (Eg, a citizen hasn't spawned yet, or one was killed).
There would potentially have a house that is not counted.